### ISSUE TITLE
Fan Control not working on all but 1 fan

### ISSUE THREAD

**Issue by yakub200** [Rem0o/FanControl.Releases#2183]
Fan Control doesnt work on any of my fans but 1, which doesnt make sense, since its the same fans as the others
![Screenshot 2023-12-08 161339](https://github.com/Rem0o/FanControl.Releases/assets/153334205/bef316c1-64e9-4ee0-8ede-40e62b736e94)

**Comment by Rem0o** [comment]
It might depend how those fans are configured in your BIOS, like if they are set to PWM vs DC and so on.

**Comment by yakub200** [comment]
I tried switch to DC and PWM and both have the same problem still, I can only control 1 fan, do you think its a problem with the sensor or something? because Fan Control things they are at 0RPM when I can see they are spinning
![Screenshot 2023-12-10 120321](https://github.com/Rem0o/FanControl.Releases/assets/153334205/be489359-7518-4b46-b1f6-696413701e06)

**Comment by Rem0o** [comment]
Are your fans actually plugged into separate motherboard headers?

**Comment by brunoosti** [comment]
I'm having the same issue. My case fans are all connected to one header on the MoBo. All fans (CPU and case) are DC and are set as so in the BIOS (with smart or constant voltage control). Inside FanCotrol those "System Fan 5" canse fans, as labled by the software, cant be controlled. The BIOS  labels this header ias #4. Maybe be because of the pump conector (witch is in use)?
My motherboard is a MSI MPG B550 GAMING PLUS. 
Thank you for the software! : )

**Comment by Rem0o** [comment]
@brunoosti 

if your fans are all connected to 1 header, then you will only have 1 "control" card that actually does something.  It's 1 control per header, not per fan.

**Comment by brunoosti** [comment]
Yes, of course, that makes complete sense. But the system fan 5 card does not change any of the 4 case fans speed even if I set the slider to 100%. I expected, as you explained, all fans would run full speed. 
CPU and VGA fans controls work as expected.

**Comment by Rem0o** [comment]
Things you can try.

Change header mode in bios. 
Use a different header.

**Comment by brunoosti** [comment]
I've disabled "smart" fan speed in bios with no success. Could this be because of the fans being DC instead PWM? Though the MoBo controles them just fine using DC smart fan mode. 
Eventually will try connecting to a different header. 
Thanks for the help!

**Comment by brunoosti** [comment]
What I've tested: 
- Only one fan connected to sys_fan4.
- Different header (sys_fan3) for all four case fans.
No success... : (. My water cooler CPU fans are PWM and case fans DC. Could the MoBo working with both interfere in how the software is able to adjust each header speed? 
Bios cand control both and monitors RPM just fine, as I've said before.
